src/unitful.jl

```diff
@@ -0,0 +1,277 @@
+# previously https://github.com/jw3126/UnitfulRecipes.jl
+# authors: Benoit Pasquier (@briochemc) - Jan Weidner (@jw3126)
+
+module UnitfulRecipes
```

Comment:
I assume this is a 1:1 copy of UnitfulRecipes?

Reply:
I've made some slight syntax changes, fixed https://github.com/jw3126/UnitfulRecipes.jl/issues/79, changed some imports, but this should be really close to `UnitfulRecipes@master`. Anyway the tests were also copied and not modified (apart from some renames for consistency with others `Plots` tests, and some additions for https://github.com/jw3126/UnitfulRecipes.jl/issues/79). So we should be covered by CI which runs all the `UnitfulRecipes` tests ...